src/wp-includes/rest-api/endpoints/class-wp-rest-block-types-controller.php

```diff
@@ -0,0 +1,549 @@
+	 *
+	 * @since 5.5.0
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'block-type',
+			'type'       => 'object',
+			'properties' => array(
+				'title'            => array(
+					'description' => __( 'Title of block type.' ),
+					'type'        => 'string',
+					'default'     => '',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'name'             => array(
+					'description' => __( 'Unique name identifying the block type.' ),
+					'type'        => 'string',
+					'default'     => '',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'description'      => array(
+					'description' => __( 'Description of block type.' ),
+					'type'        => 'string',
+					'default'     => '',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'icon'             => array(
+					'description' => __( 'Icon of block type.' ),
+					'type'        => array( 'string', 'null' ),
+					'default'     => null,
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'attributes'       => array(
+					'description'          => __( 'Block attributes.' ),
+					'type'                 => array( 'object', 'null' ),
+					'properties'           => array(),
+					'default'              => null,
+					'additionalProperties' => array(
+						'type' => 'object',
+					),
+					'context'              => array( 'embed', 'view', 'edit' ),
+					'readonly'             => true,
+				),
+				'provides_context' => array(
+					'description'          => __( 'Context provided by blocks of this type.' ),
+					'type'                 => 'object',
+					'properties'           => array(),
+					'additionalProperties' => array(
+						'type' => 'string',
+					),
+					'default'              => array(),
+					'context'              => array( 'embed', 'view', 'edit' ),
+					'readonly'             => true,
+				),
+				'uses_context'     => array(
+					'description' => __( 'Context values inherited by blocks of this type.' ),
+					'type'        => 'array',
+					'default'     => array(),
+					'items'       => array(
+						'type' => 'string',
+					),
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'supports'         => array(
+					'description' => __( 'Block supports.' ),
+					'type'        => 'object',
+					'default'     => array(),
+					'properties'  => array(),
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'category'         => array(
+					'description' => __( 'Block category.' ),
+					'type'        => array( 'string', null ),
```

Comment:
```suggestion
					'type'        => array( 'string', 'null' ),
```

And repeat for the other instances.

Reply:
I found 5 more places where this was an issue - https://github.com/WordPress/wordpress-develop/pull/317/commits/30053466afbc62ad11b238ba515328bfaa59cfdf